base/special/bessel.jl

```diff
@@ -147,6 +147,35 @@ for jy in ("j","y"), nu in (0,1)
     end
 end
 
+"""
+    besselj0(x)
+
+Bessel function of the first kind of order 0, ``J_0(x)``.
+"""
+function besselj0(x) end
```

Comment:
This will define a new method for `besselj0` returning `nothing` which we probably don't want. Should be `besselj0(x)` without the `function` and `end`. Same with the other `bessel*` and `erf*` ones below.